agents/sorenson_dlm/dlm_agent.py

```diff
@@ -0,0 +1,29 @@
+#script to log and control the Sorenson DLM power supply, for heaters
```

Comment:
The DLM Agent is a new and different Agent, it should probably be reviewed separately and not committed to this branch.